doc/changelog.md

```diff
@@ -255,18 +396,29 @@
   shot in a shots-based execution mode.
   [(#731)](https://github.com/PennyLaneAI/catalyst/pull/731)
 
+* The Catalyst frontend now supports Python logging through PennyLane's `qml.logging` module.
+  [(#660)](https://github.com/PennyLaneAI/catalyst/pull/660)
+
+* Catalyst now performs a stricter validation of the wire requirements for devices. In particular,
+  only integer, continuous wire labels starting at 0 are allowed.
+  [(#784)](https://github.com/PennyLaneAI/catalyst/pull/784)
+
+* Catalyst no longer disallows quantum circuits with 0 qubits.
+  [(#784)](https://github.com/PennyLaneAI/catalyst/pull/784)
+
 * Added support for IsingZZ gate in Catalyst frontend. Previously, the IsingZZ gate would be
   decomposed into a CNOT and RZ gates. However, this is not needed as the PennyLane-Lightning
   simulator supports this gate.
   [(#730)](https://github.com/PennyLaneAI/catalyst/pull/730)
 
-* Can now compile functions that have been annotated with return type
-  annotations.
+* Functions that have been annotated with return type
+  annotations will now correctly compile with `@qjit`.
   [(#751)](https://github.com/PennyLaneAI/catalyst/pull/751)
 
-* Refactored `vmap`,`qjit`, `mitigate_with_zne` and gradient decorators in order to follow
+* The `vmap`,`qjit`, `mitigate_with_zne` and gradient decorators now follow
   a unified pattern that uses a callable class implementing the decorator's logic. This
-  prevents having to excessively define functions in a nested fashion.
+  prevents having to excessively define functions in a nested fashion, and allows them
+  to be used as decorators.
```

Comment:
Tempted to say this is an internal change, at least the first part (excessively define functions in a nested fashion), the second part about enabling the same usage pattern across all decorators is user-facing, but the entry doesn't describe what that pattern is.

Essentially, it boils down to being able to supply keyword arguments to the decorator without the need for partial, which is quite convenient. So all our decorators allow both of:
- `@decorator`
- `@decorator(param1 = ...)`

Reply:
done!